build/qnap/shared/unified-hifi-control.sh

```
@@ -1,95 +1,78 @@
-            exit 1
-        fi
-        ;;
-    *)
-        echo "Usage: $0 {start|stop|restart|status}"
+  restart)
+    $0 stop
+    $0 start
+    ;;
+
+  status)
+    if [ -f $PIDF ] && kill -0 $(cat $PIDF) 2>/dev/null; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)